### DISCUSSION TITLE
What is the best way to clone or duplicate a single index in a instance including settings, documents, etc

### DISCUSSION THREAD

**Discussion by sheldonj** [meilisearch/product#661] [Feedback & Feature Proposal]
I was wondering what the best approach is to CLONE or DUPLICATE a single index in an instance to another index in the same instance. This would be similar to SWAP but wouldn't delete the existing index or require there to be a target instance established.

**Comment by macraig** [comment]
Hi @sheldonj, thanks for your feedback! I've transferred the discussion to our product section as we currently don't have a way to clone or duplicate an index within an instance. For now, the only thing you can do as a workaround is to resend all your data and settings to a new index.
